source/connect/tableau.txt

```diff
@@ -10,6 +10,9 @@ Connect from Tableau Desktop
    :depth: 1
    :class: singlecol
 
+The `MongoDB Connector for BI <https://www.mongodb.com/download-center?jmp=tbl#bi-connector>`_
+is a named connector in `Tableau <https://www.tableau.com/>`_. 
+
```

Comment:
Let me know if we should add more introductory content here or something to segue into the auth stuff like: 

"The connector can be configured to accept connections from Tableau with or without authentication or TLS/SSL."